tokens.json

```diff
@@ -0,0 +1 @@
+{"tokens":[]}
```

Comment:
As I understand, this is the file with the personal user information which should be not published on GitHub even if it's empty. Will the application recreate it if it suddenly gets deleted?